tests/foreman/cli/test_report.py

```diff
@@ -23,67 +23,63 @@
 from robottelo import ssh
 from robottelo.cli.base import CLIReturnCodeError
 from robottelo.cli.report import Report
-from robottelo.test import CLITestCase
 
 
-class ReportTestCase(CLITestCase):
-    """Test class for Reports CLI. """
+@pytest.fixture(scope='module', autouse=True)
+def run_puppet_agent():
+    """Retrieves the client configuration from the puppet master and
+    applies it to the local host. This is required to make sure
+    that we have reports available.
+    """
+    ssh.command('puppet agent -t')
 
-    def setUp(self):
-        super().setUp()
-        self.run_puppet_agent()
 
-    def run_puppet_agent(self):
-        """Retrieves the client configuration from the puppet master and
-        applies it to the local host. This is required to make sure
-        that we have reports available.
-        """
-        ssh.command('puppet agent -t')
+@pytest.mark.tier1
```

Comment:
not sure if this test is needed, we don't assert on anything here + Report.list() is run in both remaining tests, maybe a chance for removal

Reply:
makes sense. removed the test.

Reply:
this has been resolved